clearmash/pipeline-spec.yaml

```diff
@@ -149,3 +159,23 @@ entities-delete:
         id-field: item_id
         source: clearmash
         delete-all-input: true
+
+geocode:
```

Comment:
this pipeline should be part of a common pipeline spec file, it's not specific to clearmash